signac_dashboard/templates/cards/navigator.html

```diff
@@ -0,0 +1,24 @@
+<table class="table">
+  {% for key, ((plink, plabel), (nlink, nlabel)) in job_nav %}
```

Comment:
...and apply this below, too.
```suggestion
  {% for key, ((previous_link, previous_label), (next_link, next_label)) in job_nav %}
```